src/components/editor/Bindings/OnIncompatibleSchemaChange/Form.tsx

```diff
@@ -71,7 +73,7 @@ function Form({ bindingIndex = -1 }: OnIncompatibleSchemaChangeProps) {
                             id:
                                 err === 'no binding'
                                     ? 'updateBinding.error.noBinding'
-                                    : 'incompatibleSchemaChange.update.error',
+                                    : 'specPropEditor.update.error',
```

Comment:
We had a lot of repeated messages so starting to just share the ones that are easy to share.